test/storage_test.rb

```diff
@@ -0,0 +1,40 @@
+require 'test_helper'
+require 'ttnt/storage'
+
+class StorageTest < TTNT::TestCase
+  def setup
+    @storage_file = "#{@repo.workdir}/.ttnt"
+    FileUtils.rm_rf(@storage_file)
```

Comment:
Fixed in this too: https://github.com/Genki-S/ttnt/commit/1ec9320e6d6dff2aaa6e1d42fe42807477255e2c